tests/meta/workflow/mutant/test_mutant_quality_controller.py

```diff
@@ -0,0 +1,143 @@
+from pathlib import Path
+from cg.meta.workflow.mutant.quality_controller.models import (
+    QualityMetrics,
+    QualityResult,
+    SampleQualityResult,
+)
+from cg.meta.workflow.mutant.quality_controller.quality_controller import QualityController
+from cg.store.models import Case, Sample
+
+
+def test_quality_control_case_qc_pass(
+    case_qc_pass: Case,
+    quality_controller: QualityController,
+    mutant_analysis_dir_case_qc_pass: Path,
+    mutant_results_file_path_qc_pass: Path,
+):
+    # GIVEN a case object and a corresponding case_results_file_path
```

Comment:
```suggestion
    # GIVEN a case object passing qc and a corresponding case_results_file_path
```

Reply:
For this file in general, could we get some more info on what is being tested? The "no error is thrown" is not telling me much of whether the expected behaviour was carried out. Also, given that we are in many instances returning None instead of raising errors, there might be additional checks which we need to carry out.